ui/app/components/mount-backend-form.ts

```diff
@@ -35,7 +34,7 @@ import type { ApiError } from '@ember-data/adapter/error';
  *
  */
 
-type MountModel = SecretsEngineForm | AuthEnableModel;
```

Comment:
If the mount view is fully migrated now, does that mean we can delete this transform: https://github.com/hashicorp/vault/blob/main/ui/app/transforms/mount-visibility.js#L14? Or are you holding off until the mount model is deleted